src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```
@@ -95,7 +95,7 @@
     <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel" Version="2.2.0" />
     <PackageReference Include="Microsoft.Extensions.Azure" Version="1.1.1" />
     <PackageReference Include="Microsoft.Extensions.Http" Version="2.2.0" />
-    <PackageReference Include="Microsoft.Azure.DurableTask.ApplicationInsights" Version="0.1.0" />
+    <PackageReference Include="Microsoft.Azure.DurableTask.ApplicationInsights" Version="0.1.1" />
```

Any reason not to do this?
This is what we do for DTF.Core and DTFx.AzureStorage

```suggestion
    <PackageReference Include="Microsoft.Azure.DurableTask.ApplicationInsights" Version="0.1.*" />
```

---

Nope, no reason not to do that. Good point! I'll update it with `0.1.*`